src/main/java/org/candlepin/subscriptions/ApplicationProperties.java

```diff
@@ -165,4 +165,12 @@
      */
     private Duration metricLookupRangeDuration = Duration.ofHours(24L);
 
+    /**
+     * Latency offset: how many minutes in the past you would like the metering job latency (window) to occur.
```

Comment:
```suggestion
     * Latency offset: how far back to offset the hourly tally window.
```
?

Reply:
I sill like mine better, but okay.

Reply:
Open to other opinions too, @mstead @lindseyburnett @awood preferences? or suggestions?

Reply:
Bueller?